src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -172,33 +172,69 @@ then you have to download the database driver and put it at: *PROACTIVE_HOME/add
 
-The *second method* implies that you have a remote access to Compute Hosts (e.g. SSH access) and you want to start and stop ProActive Nodes
-by launching commands remotely on Compute Hosts. For instance, it can be useful when a virtual machine needs to be deployed prior to launching a ProActive Node.
+- the local Server Host (i.e., the machine installed the ProActive Scheduler)
+- remote compute servers
+- Cloud Platforms (e.g., Amazon Web Services, OpenStack, GoogleComputeEngine)
+- an existing cluster managed by another scheduler (e.g., SLURM, LSF, PBS)
+
+A node source is defined by two parts of configurations: <<_glossary_node_source_infrastructure,*Node Source Infrastructure*>> and <<_glossary_node_source_policy,*Node Source Policy*>>.
+Each *Node Source Infrastructure* defines the deployment mechanism for a specific kind of computational resources.
+To support the deployment of these various computational resources, the ProActive Scheduler provides a set of supported <<_node_source_infrastructure, Node Source Infrastructures>>.
+The *Node Source Policy* controls the resource usage. See the section <<_control_the_resource_usage>> for more details.
+This section concentrates on the deployment mechanism of ProActive Nodes which is defined by the Node Source Infrastructure.
+
+When you connect the Compute Hosts to the ProActive Scheduler through the *first method*, a node source of <<_default_infrastructure_manager,*Default Infrastructure*>> is automatically created.
+Through the *second method*, you just need to choose the proper node source infrastructure and fill out its required parameters.
+//The *second method* supports all the various kinds of computational resources by using different <<_node_source_infrastructure,*node source infrastructures*>> to create Node Sources.
 
 [[run_node_manually]]
 === Deploy ProActive Nodes manually
+Let's take a closer look at the first method described above. To deploy a ProActive Node from the Compute Host, you have three ways to do it:
+
+- using <<proactive_node,*proactive-node*>> command
+- using <<node_jar,*node.jar*>>
+- using <<proactive_node_autoupdate,*proactive-node-autoupdate*>> command
 
 [[proactive_node]]
 ==== Using proactive-node command
 
-Let's take a closer look at the first method described above. To deploy a ProActive Node from the Compute Host you need
-to run the following command
+To use `proactive-node` command, you need to unpack the release archive of the ProActive Scheduler on the Compute Host and run the following command
 ----
 $ PROACTIVE_HOME/bin/proactive-node -r pnp://SCHEDULER_ADDRESS:64738
 ----
-where `-r` option is used to specify the URL of the *Resource Manager*.
+where `-r` option is used to specify the URL of the <<_glossary_resource_manager,*Resource Manager*>>.
 You can find this URL in the output of the <<_run_the_proactive_scheduler,ProActive Scheduler>> (the Resource Manager URL).
```

Comment:
<<_run_the_proactive_scheduler,ProActive Server command>>

Please add a sample output of the server command describing which url is requested (people often make mistakes and use http urls)